--- page.rb	2014-09-28 05:34:32.000000000 +0000
+++ page.rb	2014-09-28 08:49:37.309980480 +0000
@@ -128,7 +128,19 @@
     #
     # Returns nothing.
     def write(dest)
-      path = destination(dest)
+      if site.config['CFE_OUTPUT'] == true && site.config['CFE_DIR'] !=''
+        dest = File.join(@base, site.config['CFE_DIR'])
+           if (self.data['alias'] !=nil)
+              path = File.join(dest, CGI.unescape(self.data['alias']))
+           else
+              path = File.join(dest, CGI.unescape(self.url))
+           end   
+        path = File.join(path, "index.html") if self.data['alias'] =~ /\/$/
+        
+      else
+        # use standard  jekyll
+        path = destination(dest)
+      end
       FileUtils.mkdir_p(File.dirname(path))
       File.open(path, 'w') do |f|
         f.write(self.output)

